src/test/FuzzerImpl.cpp

```diff
@@ -688,17 +763,87 @@ TransactionFuzzer::initialize()
         genOffersForPair(ASSET_B, ASSET_C, {13, 2, 14, 3, 15}, ltx);
         genOffersForPair(ASSET_C, ASSET_D, {6, 3, 7, 4, 8}, ltx);
 
-        // construct transaction
-        auto tx = createFuzzTransactionFrame(mSourceAccountID, ops,
-                                             mApp->getNetworkID());
+        attemptToApplyOps(ltx, mSourceAccountID, ops, *mApp);
+
+        ltx.commit();
+    }
+
+    {
+        LedgerTxn ltx(ltxOuter);
 
-        tx->attemptApplication(*mApp, ltx);
+        xdr::xvector<Operation> ops;
 
-        if (tx->getResultCode() != txSUCCESS)
+        // Reduce account balances so that fuzzing has a better change
+        // of exercising limits.
+        for (uint8_t i = 0; i < FuzzUtils::NUMBER_OF_PREGENERATED_ACCOUNTS; ++i)
         {
-            throw std::runtime_error("Error while initializing ledger state");
+            PublicKey account;
+            FuzzUtils::setShortKey(account, i);
+
+            // Reduce "account"'s native balance by paying the root.
+            LedgerKey accountKey;
```

Comment:
Look around the transaction subsystem to see how we do things. Be idiomatic: there are helpers for everything that we do often (mostly in transaction/TransactionUtils.cpp). In this case, `loadAccount` does all the work in 784,787.

Reply:
Now using `loadAccount()`.